File: billing_ui_app/src/app/core/models/company-account/funders/client-funder-model.ts
export interface ClientFunderModel {
    id: number;
    funderId: number;
    funderName: string;
    funderType: number;
    serviceLines: FunderServiceLine[];
    referringProviderRequiredOnClaim: boolean;
    startDate: Date;
    endDate: Date | null;
    isActive: boolean;
    billingProviderOptionId: number | null;
}

export interface FunderServiceLine {
    mappingId: number;
    name: string;
    sequence: string;
    serviceId: number;
    billingProviderOptionId: number;
}

File: Microservices/BillingService/BillingService.Domain/Models/Clients/ClientFunderModel.cs
using BillingService.Domain.Models.Funders;
using Rethink.Services.Common.Enums.BH;
using System;
using System.Collections.Generic;

namespace BillingService.Domain.Models.Clients
{
    public class ClientFunderModel
    {
        public int Id { get; set; }
        public int FunderId { get; set; }
        public string FunderName { get; set; }
        public FunderType FunderType { get; set; }
        public List<FunderServiceLineModel> ServiceLines { get; set; }
        public bool ReferringProviderRequiredOnClaim { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public bool IsActive { get; set; }
        public int? BillingProviderOptionId { get; set; }
    }
}